nextflow.config

```diff
@@ -231,13 +231,13 @@ dag {
 }
 
 manifest {
-    name            = 'nf-core/staramr'
-    author          = """Eric Marinier"""
-    homePage        = 'https://github.com/nf-core/staramr'
+    name            = 'phac-nml/staramrnf'
+    author          = """Steven Sutcliffe"""
```

Comment:
You can use `and` to include more than one author. Since this was started by Eric could you include `and Eric Marinier` in here?